api/rest/restapi.go

```diff
@@ -585,6 +585,62 @@ func (api *API) allocationHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// defined groups / aliases for filtering
+var filterGroups = map[string][]string{
+	"error":  {"error", "pin_error", "unpin_error", "cluster_error"},
```

Comment:
`error` and `queued` shouldn't be included in the lists of actual statuses as there is no associated `api.TrackerStatus` with those values.